usr/sbin/orjail

```diff
@@ -421,7 +426,7 @@ done
 ! [ 0 = $err ] && exit $err
 
 # check if network namespace already exists
-ip netns list | grep -e \\b"$NAME"\\b  &> /dev/null
+eno ip netns list | grep -e \\b"$NAME"\\b
```

Comment:
this will introduce a bug (try to use `-k`), should probably be `ip netns list | eno grep blabla`